lib/services/plan_payment_service.ts

```diff
@@ -0,0 +1,23 @@
+import { Stripe } from 'stripe';
+
+export interface PlanPaymentRequestBody {
+  /** The promotion codes for coupon */
+  coupon?: string;
+
+  /** The payment method on stripe */
+  paymentMethodId: string;
+
+  /** The ID of the application. */
+  appId?: string;
+
+  /** The check option plan for user by year or month. */
+  yearly?: boolean;
```

Comment:
If attribute or variable is boolean use `is...`.
Also please be more specific.
```suggestion
  isYearlyPayment?: boolean;
```

Reply:
I updated